src/pages/games/SpellingLetters.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Suspense } from "react";
+import Head from 'next/head';
+import Layout from '@/components/Layout/layout';
+import SpellingLetters from '@/features/game/SpellinLetters/SpellingLetters';
+
+const SpellingLettersPage = () => {
+    return (
+        <Layout>
+            <SpellingLetters />
```

Comment:
instead of <SpellingLetters /> why not <ModelCamera /> ?

Reply:
Because SpellingLetters will hold the entire page of the game. ModelCamera is just a component inside the SpellingLetters game page